keyboards/splitkb/halcyon/kyria/rev4/halconf.h

```diff
@@ -0,0 +1,11 @@
+// Copyright 2024 splitkb.com (support@splitkb.com)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#if HAL_USE_SPI == TRUE
+#    define SPI_USE_WAIT TRUE
+#    define SPI_SELECT_MODE SPI_SELECT_MODE_PAD
+#endif
```

Comment:
Preferable to enable these subsystems by default (they are not enabled anywhere internally), that way users don't have to go digging for why their peripherals are not working.
```suggestion
#define HAL_USE_I2C TRUE
#define HAL_USE_SPI TRUE
#define SPI_USE_WAIT TRUE
#define SPI_SELECT_MODE SPI_SELECT_MODE_PAD
```